### PR TITLE
[@mantine/dates] Add paginateBy property for Range Calendar

### PR DESCRIPTION
**Pull request by LarsBehrenberg**
Hi, this is my suggestion for the Range calendar to specify how many months are scrolled at a time. Currently when clicking to next or previous months, the amount of months scrolled is decided by the amountOfMonths property. This update would allow the user to specify how many months are "scrolled" at a time. 

PS: Happy to discuss and thanks so much for the great project! 🙏

### CODE REVIEW

File: src/mantine-dates/src/components/CalendarBase/CalendarBase.tsx [hunk 1]
```diff
@@ -96,6 +99,7 @@ export const CalendarBase = forwardRef<HTMLDivElement, CalendarBaseProps>(
       onMonthChange,
       locale,
       amountOfMonths = 1,
+      paginateBy,
```
Comment (rtivital) [inline src/mantine-dates/src/components/CalendarBase/CalendarBase.tsx:102]: It should be `paginateBy = amountOfMonths` to assign default value

**Review by rtivital — CHANGES_REQUESTED** [review]
Please fix errors to merge this PR

**Comment by rtivital** [comment]
@LarsBehrenberg Please fix errors

**Comment by LarsBehrenberg** [comment]
@rtivital fixed!

**Comment by rtivital** [comment]
Thanks!

**Pull request merged**